mne/preprocessing/maxwell.py

```diff
@@ -534,18 +555,27 @@ def maxwell_filter(raw, origin='auto', int_order=8, ext_order=3,
                 # first position in this interval is the same as last of the
                 # previous interval)
                 if trans is not None:
-                    S_decomp, pS_decomp, reg_moments, n_use_in = \
-                        _get_this_decomp_trans(trans, t=rel_times[rel_start])
+                    S_decomp, S_decomp_full, pS_decomp, reg_moments, \
+                        n_use_in = _get_this_decomp_trans(
+                            trans, t=rel_times[rel_start])
 
                 # Determine multipole moments for this interval
                 mm_in = np.dot(pS_decomp[:n_use_in],
                                orig_data[:, rel_start:rel_stop])
 
                 # Our output data
                 if not st_only:
+                    if reconstruct == 'in':
+                        proj = S_recon.take(reg_moments[:n_use_in], axis=1)
+                        mult = mm_in
+                    else:
+                        assert reconstruct == 'orig'
```

Comment:
This is an interesting construct I haven't seen before. Why not `elif reconstruct == 'orig'` and have an `else` at the bottom?

Reply:
This is a dev-only thing -- we should only ever have the two options. So I use this kind of like an "active comment" that we know this should be the case. I find it also prevents me from making silly mistakes during development, too.